source/elm-live.js

```diff
@@ -194,9 +194,18 @@ ${indent(String(elmMake.error), 2)}
   };
 
   // Watch Elm files
-  const watcher = chokidar.watch('**/*.elm', { ignoreInitial: true, followSymlinks: false });
+  const watcher = chokidar.watch('.', {
```

Comment:
Hmm, also having second thoughts about the change from `**/*.elm` to `.`. If I understand correctly, this will watch and respond to changes from the whole directory tree, right?

Even if the response is just an instant `return`, it still burdens the filesystem with potentially lots and lots of watch hooks. Depending on the OS (and perhaps other factors) this might be a massive hit.